prow/pod-utils/downwardapi/jobspec.go

```diff
@@ -177,3 +177,15 @@ func GetRevisionFromSpec(spec *JobSpec) string {
 	}
 	return ""
 }
+
+// GetBranch returns the source code branch
+func GetBranch(spec *JobSpec) string {
+	branch := spec.Refs.BaseRef
+	if spec.Type == prowapi.PostsubmitJob || spec.Type == prowapi.BatchJob {
+		return branch
+	}
+	if len(spec.Refs.Pulls) > 0 {
+		branch = fmt.Sprintf("PR-%v", spec.Refs.Pulls[0].Number)
```

Comment:
What is this?

Reply:
This was a branch made up for PR, in order to allow `tot` to generate a build number per PR. 

I will refactor it as suggested by @cjwagner bellow.

Reply:
removed